.github/workflows/test-web-twig.yaml

```
@@ -0,0 +1,90 @@
+name: 'Web Twig: Code Quality Checks'
+
+on:
+  push:
+    paths:
+      - packages/web-twig/**
+      # Because of snapshot testing
```

The comment could be a little bit more helpful like this, I believe:

```suggestion
      # Icons are part of `web-twig` snapshots, check if they are up-to-date.
```

Or something like that, YKWIM.